Checkpoints/checkpoint2.js

```diff
@@ -0,0 +1,51 @@
+'use strict';
+
```

Comment:
Missing documentation on the assignment.
The map function is really meant to return a new array. In this case it would return an array of statements.